content/04.workflows-and-software.md

```diff
@@ -36,22 +36,21 @@ Here, we draw a distinction between two types of workflows: "research" workflows
 Many workflow systems can be used for either type, but we note cases where their properties facilitate one of these types over the other.
 
-**Using workflows without learning management systems**
-While the benefits of encoding a workflow in a workflow system are immense, the learning curve associated with implementing complete workflows in a new syntax can be daunting.
-It is possible to obtain the benefits of workflow systems without learning a workflow system.
+**Using workflows without learning workflow syntax**
+While the benefits of executing an analysis within a data-centric workflow system are immense, the learning curve associated with command-line systems can be daunting.
+It is possible to obtain the benefits of workflow systems without learning new syntax.
 Websites like Galaxy, Cavatica, and EMBL-EBI MGnify offer online portals in which users build workflows around publicly-available or user-uploaded data [@doi:10.1093/nar/gky379; @doi:10.14694/EDBK_175029; @doi:10.1093/nar/gkz1035].
-On the command line, many research groups have used workflow systems to build user-friendly pipelines that do not require learning or working with the underlying workflow software.
-These tools are specified in an underlying workflow language, but are packaged in a user-friendly command-line script that coordinates and executes the workflow.
-Rather than writing each workflow step, the user can specify data and parameters in a configuration file to customize the run.
-Some examples include the nf-core RNA-seq pipeline [@url:https://github.com/nf-core/rnaseq/; @doi:10.1038/s41587-020-0439-x], the ATLAS metagenome assembly and binning pipeline [@url:https://github.com/metagenome-atlas/atlas; @doi:10.1101/737528], the Sunbeam metagenome analysis pipeline [@url:https://github.com/sunbeam-labs/sunbeam; @doi:10.1186/s40168-019-0658-x], and two from our own lab, the dammit eukaryotic transcriptome annotation pipeline [@url:https://github.com/dib-lab/dammit] and the elvers *de novo* transcriptome pipeline [@url:https://github.com/dib-lab/elvers].
-These tools allow users to take advantage of the benefits of workflow software without needing to invest in curating and writing their own pipeline. The majority of these workflows are production-level workflows designed to execute a series of standard steps, but many provide varying degrees of customizability ranging from tool choice to parameter specification.
+On the command line, many research groups have used workflow systems to wrap one or many analysis steps (specified in an underlying workflow language) in a more user-friendly command-line application that accepts user input and executes the analysis.
+These pipeline applications allow users to take advantage of the benefits of workflow software without needing to write the workflow syntax for each analysis step.
```

Comment:
suggestion: remove "of the benefits of" => "allow users to take advantage of workflow software".

Could also add something installability here.